ansible/roles/common/defaults/main.yaml

```diff
@@ -0,0 +1,8 @@
+---
+# The directory where binaries are stored on Ansible
+# managed systems.
+bin_dir: /usr/local/bin
+
+# The directory used by Ansible to temporarily store
+# files on Ansible managed systems.
+ansible_temp_dir: /tmp/.ansible/files
```

Comment:
I think I'd like to see a tmp_dir created in pre-ansible (via mktemp) rather than do this. But that's probably just a matter of taste...